packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -220,6 +221,96 @@ describe('Grant Resolvers', (): void => {
         expect(query.edges).toHaveLength(0)
         expect(grants).toHaveLength(2)
       })
+
+      test('finalizationReason: in', async (): Promise<void> => {
+        const grants: GrantModel[] = []
+        for (let i = 0; i < 2; i++) {
```

Comment:
maybe we do 3 so that we can have an odd one out with a different status, if that makes sense?

Reply:
I did that here as well as the other tests. Also verified the returned records' fields filtered by match the filter (or don't for `notIn`). I think these assertions are in a better place now. I also wonder about not testing each filter here since they are tested in services and just making sure the service `getPage` function is called correctly...